test.txt

```diff
@@ -0,0 +1,95 @@
+
```

Comment:
nit: this file added accidentally? if you were trying to regenerate the contents you have to set the env var `REWRITE`.